src/api/verify.ts

```diff
@@ -78,14 +83,86 @@ class VerifyAPI {
       expiresAt,
       expiresIn,
       passed,
+      failureReasons,
+      _id,
     } as RadarTrackVerifiedResponse;
 
     if (options.debug) {
       trackRes.response = response;
     }
 
+    lastToken = trackRes;
+    lastTokenNow = performance.now();
+
+    if (tokenCallback) {
+      tokenCallback(trackRes);
+    }
+
     return trackRes;
   }
+
+  static async startTrackingVerified(params: RadarStartTrackingVerifiedParams) {
+    const doTrackVerified = async () => {
+      const trackRes = await this.trackVerified({});
+
+      const { interval } = params;
+
+      let expiresIn = 0;
+      let minInterval = interval;
+
+      if (trackRes) {
+        expiresIn = (trackRes.expiresIn || expiresIn);
+
+        // if expiresIn is shorter than interval, override interval
+        minInterval = Math.min(expiresIn, interval);
+      }
+
+      // re-request early to maximize the likelihood that a cached token is available
+      if (minInterval > 20) {
+        minInterval = minInterval - 10;
+      }
+
+      // min interval is 10 seconds
+      if (minInterval < 10) {
+        minInterval = 10;
+      }
+
+      if (tokenTimeoutId) {
+        clearTimeout(tokenTimeoutId);
+      }
+
+      tokenTimeoutId = setTimeout(doTrackVerified, minInterval * 1000);
+    };
+
+    doTrackVerified();
+  }
+
+  static stopTrackingVerified() {
+    if (tokenTimeoutId) {
+      clearTimeout(tokenTimeoutId);
+    }
+  }
+
+  static async getVerifiedLocationToken() {
+    const lastTokenElapsed = (performance.now() - lastTokenNow) / 1000;
+
+    if (lastToken) {
+      if (lastTokenElapsed < (lastToken.expiresIn || 0)) {
+        return lastToken;
+      }
+    }
+
+    return this.trackVerified({});
+  }
+
+  static setExpectedJurisdiction(countryCode?: string, stateCode?: string) {
+    expectedCountryCode = countryCode || null;
```

Comment:
I was getting errors because `expectedCountryCode` is defined as `| null` instead of `| undefined`. https://app.circleci.com/pipelines/github/radarlabs/radar-sdk-js/1241/workflows/a23a44d8-bbed-4f3d-a059-163a7b4407c0/jobs/1016

Reply:
I was getting errors because `expectedCountryCode` is defined as `| null` instead of `| undefined`. https://app.circleci.com/pipelines/github/radarlabs/radar-sdk-js/1241/workflows/a23a44d8-bbed-4f3d-a059-163a7b4407c0/jobs/1016

Reply:
Yeah, I think you can just define them as:
```ts
let expectedCountryCode?: string;
```